src/helpers/models.rs

```diff
@@ -0,0 +1,140 @@
+use num_bigint::BigInt;
+use serde::{Deserialize, Serialize};
+use std::fmt::{Debug, Formatter};
+use std::ops::Range;
+
+// Type aliases to indicate whether the parameter should be encrypted, secret shared, etc.
+// Underlying types are temporalily assigned for PoC.
+type CipherText = BigInt;
```

Comment:
@martinthomson - if we use some basic encryption approach like AES, what would be the right type signature to store those CipherTexts? Is it BigInt, or is it something else?

Reply:
As a buffer of bytes generally, so `Vec<u8>` or `&[u8]` depending on whether you need a reference.